Impossible-Finance-Agents/Swap-Fees/README.md

```diff
@@ -2,13 +2,13 @@
 
 ## Description
 
-This agent detects transactions where modifications in fee values of Impossible Finance V2 Pairs occurred.
+This bot detects transactions where modifications in fee values of Impossible Finance V2 Pairs occurred.
 > Check Impossible Finance V2 Pairs and Factory here https://github.com/ImpossibleFinance/impossible-swap-core
 > You can update the Factory address to V2 Factory by changing `FACTORY` value in the agent file. 
```

Comment:
```suggestion
This bot detects transactions where modifications in fee values of Impossible Finance V3 Pairs occurred.
```
Update V2 to V3 in description. Notes below can be removed after updating the factory version to V3.